src/gawronski-bartosz/Fsd.Bartek.Cs/Fsd.Bartek.Cs.Ex2/Domain/Fabric.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Fsd.Bartek.Cs2.Ex2.Domain.Animals;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fsd.Bartek.Cs2.Ex2.Domain.UI;
+
+namespace Fsd.Bartek.Cs2.Ex2.Domain
+{
+    public class Fabric
```

Comment:
Fabric to "tkanina". użyj Factory. Dodaj też info co produkuje dana fabryka. Najlepiej AnimalFactory